pkg/repository/v1/match.go

```diff
@@ -485,7 +485,7 @@ func (m *sharedRepository) processEventMatches(ctx context.Context, tx sqlcv1.DB
 
 					switch matchData.Action() {
 					case sqlcv1.V1MatchConditionActionQUEUE:
-						opt.Input = m.newTaskInput(input, matchData)
+						opt.Input = m.newTaskInput(input, matchData, nil)
```

Comment:
I think we need to pass this to this level somehow? These are steps which have been triggered in the DAG, should they not have access to the triggering payload?

Reply:
ah yeah good call, I wasn't sure how to do this but I'll figure it out